CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ### Fixed
 - in PR template spell "compatibility" correctly
 
+## [2.1.1] - 2017-07-07
```

Comment:
Currently we do the versioning on the sensu maintainers side as there is no guarantee on order of PR or issues addressed and we want to limit the number of times that you need to rebase. Also this changelog entry is in the wrong spot, please rebase and put `### Fixed` under `### [Unreleased]`